internal/ffmpeg/preview.go

```diff
@@ -5,7 +5,7 @@ import "time"
 // PreviewTimeOffset returns an appropriate time offset depending on the duration for extracting a preview image.
 func PreviewTimeOffset(d time.Duration) string {
 	// Default.
-	result := "00:00:00.001"
+	result := "00:00:00.000"
```

Comment:
This change is interesting! Did it cause any issues when viewing live/motion photos in the UI?

If I remember correctly, there were very few files that had a black still image if you specified exactly `00:00:00.000` here. But since that was a long time ago, I can't provide you with any details...

Reply:
I didn't notice any issues in the UI. Without this change though a number of photos failed to index because ffmpeg resulted in 
```
Output file is empty, nothing was encoded (check -ss / -t / -frames parameters if used)
```

This caused the check for the side car file to fail and the index to not complete. Too a while to find, as the error returned from index was not logged (which is why added the log). I couldn't find a reason why some files hit that issue and others didn't.

This is what ultimately led to my question though. Do we even need to generate a preview image on these mp4s. The original image is effectively the preview image. This would be a larger change though so left it for a future change set

Reply:
Would you be able to share some affected files for testing? I've never this happening before.

Reply:
almost all my photos have faces that I am not comfortable sharing, however I was able to find one to share. I have provided the raw image and the extracted mp4 that gives the issue https://www.dropbox.com/scl/fo/h954riwxmguqntzegq6aj/h?rlkey=jx69sw2i77r8rlzezp68gjb80&dl=0

Reply:
fwiw, I just downloaded the video portion straight from google photos, with the "Download Video" option. It also has the same issue as the one extracted from the jpg. Meaning I don't think its related to how we are doing extraction. 


And to share more detail why this is problematic. FFmpeg wouldn't produce an output file, which cause this to error https://github.com/photoprism/photoprism/blob/8b6de380d0ef4486972a5168f496e75fe81ec351/internal/photoprism/convert_image.go#L101 because the file doesn't exist on disk

Reply:
As a general observation, Google and Samsung seem to add the start frame and metadata at the END of their embedded video files - contrary to their own explicit recommendation on how you should do it. This prevents HTTP streaming with chunked encoding and makes extracting a still image overly complicated as you have to read the entire file, which can take a long time if you are using remote network storage.